src/org/labkey/test/tests/issues/IssueDomainTest.java

```diff
@@ -0,0 +1,126 @@
+package org.labkey.test.tests.issues;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.remoteapi.domain.DomainCommand;
+import org.labkey.remoteapi.domain.DomainResponse;
+import org.labkey.remoteapi.domain.GetDomainCommand;
+import org.labkey.remoteapi.domain.SaveDomainCommand;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.categories.DailyA;
+import org.labkey.test.categories.Data;
+import org.labkey.test.categories.Issues;
+import org.labkey.test.util.IssuesHelper;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+@Category({Issues.class, DailyA.class, Data.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 20)
```

Comment:
This looks like it should be a lot faster than 20 minutes. Suggest removing this annotation and letting it use the default.

Reply:
that was a copy paste, now changed to 2 mins. On my local machine total time is 1m 46 secs (which includes build time and test time), so 2 mins should be good.